### PR TITLE
updates for 0.13.0

### PR DESCRIPTION
**Pull request by CaroMac**

### CODE REVIEW

File: src/markdown-pages/about_Galasa.md [hunk 1]
```diff
@@ -3,19 +3,41 @@ path: "/about"
 title: "About Galasa"
 ---
 
-Galasa is different from other test tools. It is a deep integration test framework, enabling you to test applications that interact with z/OS (including CICS applications) without stubbing or mocking. 
+Galasa is different from other test tools; enabling deep integration testing across platforms and technologies within a DevOps pipeline, Galasa supports repeatable, reliable, agile testing at scale across your enterprise.
 
-Galasa tests are not limited to testing mainframe-specific applications - they can extend to encompass entire families of hybrid cloud applications. Most manual tests can be automated with Galasa, helping you to deliver changes to your core applications quickly and successfully. 
+### Platform integration
+The Galasa framework is built with a native knowledge of z/OS and hybrid multi-cloud, enabling end-to-end testing of applications that run on different platforms without stubbing or mocking.
```
Comment (hobbit1983) [inline src/markdown-pages/about_Galasa.md:9]: "native knowledge of z/OS and cloud native platforms"

File: src/markdown-pages/about_Galasa.md [hunk 2]
```diff
@@ -3,19 +3,41 @@ path: "/about"
...
 
-Galasa is different from other test tools. It is a deep integration test framework, enabling you to test applications that interact with z/OS (including CICS applications) without stubbing or mocking. 
+Galasa is different from other test tools; enabling deep integration testing across platforms and technologies within a DevOps pipeline, Galasa supports repeatable, reliable, agile testing at scale across your enterprise.
 
-Galasa tests are not limited to testing mainframe-specific applications - they can extend to encompass entire families of hybrid cloud applications. Most manual tests can be automated with Galasa, helping you to deliver changes to your core applications quickly and successfully. 
+### Platform integration
+The Galasa framework is built with a native knowledge of z/OS and hybrid multi-cloud, enabling end-to-end testing of applications that run on different platforms without stubbing or mocking. 
+
+### Technology integration
+Galasa is fully extensible, supporting integration across technologies by using an API that interacts with multiple test tools, including JMeter, Selenium, JCL and 3270 screens and more.
```
Comment (hobbit1983) [inline src/markdown-pages/about_Galasa.md:12]: Galasa is fully extensible, galasa managers allow it to interact with any test technology such as ....

File: src/markdown-pages/about_Galasa.md [hunk 3]
```diff
@@ -3,19 +3,41 @@ path: "/about"
...
 
-Galasa tests are not limited to testing mainframe-specific applications - they can extend to encompass entire families of hybrid cloud applications. Most manual tests can be automated with Galasa, helping you to deliver changes to your core applications quickly and successfully. 
+### Platform integration
+The Galasa framework is built with a native knowledge of z/OS and hybrid multi-cloud, enabling end-to-end testing of applications that run on different platforms without stubbing or mocking. 
+
+### Technology integration
+Galasa is fully extensible, supporting integration across technologies by using an API that interacts with multiple test tools, including JMeter, Selenium, JCL and 3270 screens and more. 
+
+### Pipeline integration
+Galasa provides a REST endpoint that can be called from any CI/CD pipeline, for example, a Jenkins pipeline. Running Galasa as part of your pipeline enables fast, reliable testing at scale - Galasa can run thousands of tests in parallel.
```
Comment (hobbit1983) [inline src/markdown-pages/about_Galasa.md:15]: should we mention that we supply a Jenkins plugin?

File: src/markdown-pages/about_Galasa.md [hunk 4]
```diff
@@ -3,19 +3,41 @@ path: "/about"
...
+The Galasa framework is built with a native knowledge of z/OS and hybrid multi-cloud, enabling end-to-end testing of applications that run on different platforms without stubbing or mocking. 
+
+### Technology integration
+Galasa is fully extensible, supporting integration across technologies by using an API that interacts with multiple test tools, including JMeter, Selenium, JCL and 3270 screens and more. 
+
+### Pipeline integration
+Galasa provides a REST endpoint that can be called from any CI/CD pipeline, for example, a Jenkins pipeline. Running Galasa as part of your pipeline enables fast, reliable testing at scale - Galasa can run thousands of tests in parallel. 
+
+### Enterprise integration
+Galasa is not simply about providing the ability to write a test and run it in automation.  Galasa's test catalog enables you to understand and manage your test corpus, making it easy to schedule the right tests to run at the right time. Test results and artifacts are stored in a single repository, making reporting and problem determination simple when combined with native Kibana and Grafana dashboards.
```
Comment (hobbit1983) [inline src/markdown-pages/about_Galasa.md:18]: corpus or catalog?

File: src/markdown-pages/about_Galasa.md [hunk 5]
```diff
@@ -3,19 +3,41 @@ path: "/about"
...
+
+Using hybrid cloud, enterprises can host applications and data on the platforms that deliver the best combination of efficiency, security, and performance. However, a mix of platforms and technologies makes end-to-end integration testing complicated.
+
+Galasa simplifies testing in such an environment. The following diagram shows an example of how you can use Galasa to test a hybrid cloud application:  
+
+![Galasa in hybrid cloud](galasa-hybrid-cloud.svg)
+
+This sophisticated solution requires end-to-end integration testing of an application that runs on different platforms (z/OS and Cloud) and uses different technologies (a 3270 emulator, JCL batch job and Selenium Web Driver). 
+
+The Galasa test can be invoked from an IDE or as part of a CI/CD pipeline (a Jenkins pipeline in this scenario). The Galasa framework initializes the test environment, creates valid test data, runs the test and validates the test output. As all test results and artifacts are stored in one location, it's easy to generate reports and diagnose the cause of any failures.
```
Comment (hobbit1983) [inline src/markdown-pages/about_Galasa.md:34]: should we mention that the galasa ecosystem is a cloud native application exposing a sequence of micro services that are used to manage the set of running tests?

File: src/markdown-pages/docs/ecosystem/ecosystem-architecture.md [hunk 6]
```diff
@@ -3,14 +3,14 @@ path: "/docs/ecosystem/architecture"
 title: "Ecosystem Architecture"
 ---
 
-The Galasa ecosystem contains all of the servers and monitors that are required to run Galasa tests in an automated environment or as part of a DevOps pipeline. The following diagram provides a high-level representation of the Galasa ecosystem architecture:<br><br> ![Galasa ecosystem architecture:](ecosystem-overview-final.svg)
+The Galasa ecosystem contains all of the servers and monitors that are required to run Galasa tests in an automated environment or as part of a DevOps pipeline. The following diagram provides a high-level representation of the Galasa ecosystem architecture:<br><br> ![Galasa ecosystem architecture:](ecosystem-overview.svg)
```
Comment (hobbit1983) [inline src/markdown-pages/docs/ecosystem/ecosystem-architecture.md:6]: lets leave the sentence at in an automated environment.

Comment (hobbit1983) [inline src/markdown-pages/docs/ecosystem/ecosystem-architecture.md:6]: contains all of the micro services and monitors ....

**Review by hobbit1983 — CHANGES_REQUESTED** [review]

File: src/markdown-pages/about_Galasa.md (hunk 1 above)
Comment (CaroMac) [reply src/markdown-pages/about_Galasa.md]: have updated

File: src/markdown-pages/about_Galasa.md (hunk 2 above)
Comment (CaroMac) [reply src/markdown-pages/about_Galasa.md]: updated

File: src/markdown-pages/about_Galasa.md (hunk 3 above)
Comment (CaroMac) [reply src/markdown-pages/about_Galasa.md]: have added this information

File: src/markdown-pages/about_Galasa.md (hunk 4 above)
Comment (CaroMac) [reply src/markdown-pages/about_Galasa.md]: changed it to read tests rather than test corpus

File: src/markdown-pages/about_Galasa.md (hunk 5 above)
Comment (CaroMac) [reply src/markdown-pages/about_Galasa.md]: I have added that line in the Galasa ecosystem doc and added a link to that topic from the About section.

File: src/markdown-pages/docs/ecosystem/ecosystem-architecture.md (hunk 6 above)
Comment (CaroMac) [reply src/markdown-pages/docs/ecosystem/ecosystem-architecture.md]: done

Comment (CaroMac) [reply src/markdown-pages/docs/ecosystem/ecosystem-architecture.md]: done

**Review by Michael-Baylis — APPROVED** [review]

**Review by hobbit1983 — APPROVED** [review]

**Pull request merged**